insights/parsers/ld_library_path.py

```diff
@@ -64,3 +64,46 @@ def parse_content(self, content):
             raise SkipComponent("LD_LIBRARY_PATH not set.")
 
         self.extend(llds)
+
+
+@parser(Specs.ld_library_path_global_conf)
+class GlobalLdLibraryPathConf(Parser):
+    """
+    Class to parse the datasource ``ld_library_path_global_conf`` output.
+
+    Sample output of datasource looks like::
+        export_files: /etc/environment,/etc/env.d/test.conf,/root/.bash_profile
+        unset_files: /etc/profile
+
+    ``export_files`` line is a list of files that define the global LD_LIBRARY_PATH environment.
+    ``unset_files`` line is a list of files that unset the global LD_LIBRARY_PATH environment.
+
+    Examples:
+        >>> type(global_ld_library_path_conf)
+        <class 'insights.parsers.ld_library_path.GlobalLdLibraryPathConf'>
+        >>> global_ld_library_path_conf.export_files
+        '/etc/environment,/etc/env.d/test.conf,/root/.bash_profile'
+        >>> global_ld_library_path_conf.unset_files
+        '/etc/profile'
+
+    Attributes:
+        export_files(str): a list of files that define the global LD_LIBRARY_PATH environment.
+        unset_files(str): a list of files that unset the global LD_LIBRARY_PATH environment.
```

Comment:
With introducing this `unset_files`, I have such questions, e.g. in some files (or even the same file), there are `unset` and `export` lines for the `LD_LIBRARY_PATH` variable. How does the plugins to identify which one works finally? If it's unset at last, the plugin will hit false positive.

Reply:
Talked with @wushiqinlou - we have workaround (ignore host that have ever `unset` this ENV Var in any listed files). It's good to me.